OPEN_PROBLEMS/PRESERVE_USER_PRIVACY.md

```diff
@@ -0,0 +1,109 @@
+- [DoubleRatchet](https://signal.org/docs/specifications/doubleratchet/) / [Matrix.org Olm/MegaOLM](https://gitlab.matrix.org/matrix-org/olm/blob/master/docs/olm.md#olm-a-cryptographic-ratchet)
+- [Using Sphinx to Improve Onion RoutingCircuit Construction](https://www.cypherpunks.ca/~iang/pubs/SphinxOR.pdf)
+- [Scuttlebutt Secret Handshare](https://dominictarr.github.io/secret-handshake-paper/shs.pdf)
+- [Octopus: A Secure and Anonymous DHT Lookup](https://ieeexplore.ieee.org/document/6258005)
+- [Vuvuzela: Scalable Private Messaging Resistant to Traffic Analysis](https://davidlazar.org/papers/vuvuzela.pdf)
+- [Define privacy threat model for DHTs (and other overlay P2P networks)](https://github.com/gpestana/notes/issues/3)
+- [Talek: a Private Publish-Subscribe Protocol](https://raymondcheng.net/download/papers/talek-tr.pdf)
+- [SoK: Secure Messaging](https://ieeexplore.ieee.org/document/7163029)
+- [Ricochet](https://github.com/ricochet-im/ricochet/blob/master/doc/protocol.md)
+- [Cwtch: Privacy Preserving Infrastructure for Asynchronous,Decentralized, Multi-Party and Metadata Resistant Applications](https://cwtch.im/cwtch.pdf)
+- [HOPR - privacy-preserving messaging protocol ](https://github.com/validitylabs/hopr)
+
+### Known shortcommins of existing solutions
+> What are the limitations on those solutions?
+
+Some of the known shortcommings of existing solutions are:
+
+- They don't offer protection against network analysis (it is possible to infer what the user is doing by analysing network traffic)
+- Solutions that are more resistant (not fully resistent) typically trade off bandwidth + memory for creating that protection (e.g. creating noise in the network to make it hard to distinguish valid from dummy traffic) 
+- Lack of data encryption at rest
+- Lack of complete authorization + revocation
+
+## Solving this Open Problem
+
+### What is the impact
+
+Finding a complete solution to this Open Problem will grant any human in the planet the ability to privately access or share information without leaking their intent to non-local observers other than the provider of that content. Solutions may also be able to hide said intent from the content provider.
+
+It will lead to a generation of safer applications, both consumer level and business critical, that require full control of one's data (e.g. Health Data).
+
+### What defines a complete solution?
+> What hard constraints should it obey? Are there additional soft constraints that a solution would ideally obey?
+
+Valid solutions should improve the current state of the art or offer definitive solutions for:
+
+- [ ] Two users can transfer a piece of information without any other user knowing or being able to predict: what it is, why it is being transferred, when and between whom.
+- [ ] No single central authorities are required to mediate the communication
```

Comment:
This is not incorrect but **single** ... authoriti**es** always read a bit funny.

Reply:
```suggestion
- [ ] No single central authority are required to mediate the communication
```